core/arch/arm/plat-vexpress/conf.mk

```diff
@@ -55,6 +55,11 @@ endif
 CFG_WITH_STATS ?= y
 CFG_ENABLE_EMBEDDED_TESTS ?= y
 
+ifeq ($(CFG_SCMI_SERVER),y)
+CFG_SCMI_SERVER_PRODUCT=fvp
```

Comment:
In the commit "plat-stm32mp1: support building with CFG_SCMI_SERVER=y" you're using force instead, any particular reason?

Reply:
no, but here I should have used `$(call force,...)` or `?=`. I'll fix, using `$(call force,...)`.